### ISSUE TITLE
Color is gone

### ISSUE THREAD

**Issue by timhaigh** [traviscross/mtr#73]
Every version of MTR I have compiled to run on Mac OS X appears to reflect the colour scheme of my bash profile and my terminal settings.

version 0.85 looks good with a view that is pleasing to the eye.  as soon as I  upgraded MTR to version 0.86  all i get is a grey and black monochrome output.

**Comment by sekrause** [comment]
I agree, version 0.85 looked much better because it didn't change my terminal colors.

**Comment by upa** [comment]
mtr version 0.86 installed from mac ports on 10.10.3 also withdraws colors. But, I guess it was fixed on commit https://github.com/traviscross/mtr/commit/63a1f1493bfbaf7e55eb7e20b3791fc8b14cf92d (unfortunately, it is not included on current mtr tarball on macports).

So, latest mtr on github I have compiled on 10.10.3 works fine with terminal colors.

**Comment by chrisminett** [comment]
+1, via MacPorts
